### ISSUE TITLE
Enhance Remote Background | Event Log

### ISSUE THREAD

**Issue by silversword411** [wh1te909/tacticalrmm#535]
Add colors and filters to Event Log.
Red = Error
Yellow = Warning

![2021-05-27_104358 - event viewer enhancement](https://user-images.githubusercontent.com/13395348/119847011-a11a4980-bed8-11eb-8668-a15ec8fcf3eb.png)

Checkboxes that hide/show those types (remembers your settings across clients). ie if someone unchecks info, it will be unchecked for all agents/Event Logs (they probably aren't interested in info logs)

**Comment by silversword411** [comment]
...this is a Mesh feature add isn't it.

**Comment by wh1te909** [comment]
no, everything in remote background is handled by tactical except the file browser and terminal tabs

**Comment by silversword411** [comment]
https://github.com/amidaware/tacticalrmm/issues/2204
